### PR TITLE
Added Central Bank of the Republic of Turkey provider.

### PR DESCRIPTION
**Pull request by ugurerkan**
Hello,

Created new provider for Central Bank of the Republic of Turkey. 
It's similar to the National Bank of Romania provider but not based on base currency.

CBRT is publishing currencies as quote currency in TRY. 
Eg: EUR/TRY, USD/TRY etc.

Currencies provided from and [tcmb.gov.tr/kurlar/today.xml ](http://tcmb.gov.tr/kurlar/today.xml ) updated daily based on 15:30.

### CODE REVIEW

File: src/Provider/CentralBankOfRepublicTurkeyProvider.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            throw new UnsupportedCurrencyPairException($currencyPair);
+        }
+
+        $rootAttributes = $xmlElement->attributes();
+        $date = new \DateTime((string) $rootAttributes['Date']);
+
+        foreach ($xmlElement->Currency as $currency) {
+            $currencyAttributes = $currency->attributes();
+
+            if((string) $currencyAttributes['CurrencyCode'] === $currencyPair->getBaseCurrency())
```
Comment (florianv) [inline src/Provider/CentralBankOfRepublicTurkeyProvider.php]: There should be an extra space `if (`

File: tests/Tests/Provider/CentralBankOfRepublicTurkeyProviderTest.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        $url = 'http://www.tcmb.gov.tr/kurlar/today.xml';
+        $content = file_get_contents(__DIR__ . '/../../Fixtures/Provider/CentralBankOfRepublicTurkey/cbrt_today.xml');
+
+        $provider = new CentralBankOfRepublicTurkeyProvider($this->getHttpAdapterMock($url, $content));
+        $rate = $provider->fetchRate(new CurrencyPair('EUR', 'TRY'));
+
+        $this->assertSame('3.2083', $rate->getValue());
+        $this->assertEquals(new \DateTime('2016-03-15'), $rate->getDate());
+    }
+
```
Comment (florianv) [inline tests/Tests/Provider/CentralBankOfRepublicTurkeyProviderTest.php]: There is an extra line here

File: tests/Tests/Provider/CentralBankOfRepublicTurkeyProviderTest.php [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+}
+
```
Comment (florianv) [inline tests/Tests/Provider/CentralBankOfRepublicTurkeyProviderTest.php]: There is an extra line here

**Comment by florianv** [comment]
Hello @ugurerkan, thanks for your contribution.

I will merge it once the small code style issues are fixed

**Comment by ugurerkan** [comment]
Thank you for your review @florianv, issues are fixed now.

**Pull request merged**